src/test/java/seedu/address/testutil/TypicalStudents.java

```diff
@@ -48,15 +40,19 @@
 
 
     // Manually added - Person's details found in {@code CommandTestUtil}
-    public static final Student AMY = new StudentBuilder().withName(VALID_NAME_AMY)
-            .withPhone(VALID_PHONE_AMY)
-            .withSchool(VALID_SCHOOL_AMY)
-            .withYear(VALID_YEAR_AMY)
+    public static final Student AMY = new StudentBuilder().withName(CommandTestUtil.VALID_NAME_AMY)
+            .withPhone(CommandTestUtil.VALID_PHONE_AMY)
+            .withSchool(CommandTestUtil.VALID_SCHOOL_AMY)
+            .withYear(CommandTestUtil.VALID_YEAR_AMY)
+            .withClassVenue(CommandTestUtil.VALID_VENUE_AMY)
+            .withClassTime(CommandTestUtil.VALID_TIME_AMY)
+            .withFee(CommandTestUtil.VALID_FEE_AMY)
+            .withPaymentDate(CommandTestUtil.VALID_PAYMENT_AMY)
             .build();
-    public static final Student BOB = new StudentBuilder().withName(VALID_NAME_BOB)
-            .withPhone(VALID_PHONE_BOB)
-            .withSchool(VALID_SCHOOL_BOB)
-            .withYear(VALID_YEAR_BOB)
+    public static final Student BOB = new StudentBuilder().withName(CommandTestUtil.VALID_NAME_BOB)
```

Comment:
Will you be adding them withClassVenue/ClassTime/Fee/PaymentDate for Bob?

Reply:
Yea I will be adding the admin fields when doing up the updated test cases.